charts/scaleway-cilium-hubble/templates/networkpolicies.yml

```
@@ -0,0 +1,66 @@
+{{- if .Values.networkPolicies.enabled }}
+---
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: hubble-relay
+spec:
+  description: "Allow hubble-relay to connect to the cilium network"
+  endpointSelector:
+    matchLabels:
+      k8s-app: hubble-relay
+  egress:
+    - toEndpoints:
+        - matchLabels:
+            "k8s:io.kubernetes.pod.namespace": "kube-system"
+            "k8s:k8s-app": "kube-dns"
+      toPorts:
+        - ports:
+            - port: "53"
+              protocol: ANY
+          rules:
+            dns:
+              - matchPattern: "*"
+    - toEntities:
+        - host
+        - remote-node
+      toPorts:
+        - ports:
+            - port: "4244"
+              protocol: "TCP"
+  ingress:
+    - fromEndpoints:
+        - matchLabels:
+            "k8s:k8s-app": "hubble-ui"
+---
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: hubble-ui
+spec:
+  description: "Allow Hubble UI to connect to hubble-relay & api-server"
+  endpointSelector:
+    matchLabels:
+      k8s-app: hubble-ui
+  egress:
+    - toEndpoints:
+        - matchLabels:
+            "k8s:io.kubernetes.pod.namespace": "kube-system"
+            "k8s:k8s-app": "kube-dns"
+      toPorts:
+        - ports:
+            - port: "53"
+              protocol: ANY
+          rules:
+            dns:
+              - matchPattern: "*"
+    - toEndpoints:
+        - matchLabels:
+            "k8s:k8s-app": "hubble-relay"
+      toPorts:
+        - ports:
+            - port: "4245"
+              protocol: "TCP"
+    - toEntities:
+        - kube-apiserver
+{{- end }}
```

Missing end of file line

```suggestion
{{- end }}

```